src/components/signin/components/Signin.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+
+import { 
+    Button,
+    Checkbox,
+    Form,
+    Icon,
+    Input,
+    Modal
+} from 'antd';
+
+import { SIGNIN_STRINGS } from './constants';
+
+const {
+    CIRCLE,
+    CHECKBOX_REMINDER,
+    CLASSNAME_LOGIN,
+    ICON_LOCK,
+    ICON_USER,
+    MESSAGE_1,
+    PASSWORD,
+    PLACEHOLDER_1,
+    PLACEHOLDER_2,
+    PRIMARY,    
+    SUBMIT,
+    TITLE,
+} = SIGNIN_STRINGS;
+
+const FormItem = Form.Item;
+
+const SigninForm = Form.create()(
+    class SigninForm extends React.Component {
+        render() {
+            const { form, visible, onCancel, onCreate } = this.props;
+            const { getFieldDecorator } = form;
+            return (
+                <Modal
+                    visible={visible}
+                    title={TITLE}
+                    onCancel={onCancel}
+                    onOk={onCreate}
+                >
+                    <Form>
+                        <FormItem>
+                            {getFieldDecorator("title", {
+                                rules: [{ message: { MESSAGE_1 }, required: true }],
+                            })(
+                                <Input
+                                    placeholder= { PLACEHOLDER_1 }
+                                    prefix={<Icon type={ICON_USER} />}
+                                />
+                            )}
+                        </FormItem>
+                        <FormItem>
+                            {getFieldDecorator("description")(
+                                <Input
+                                    placeholder= {PLACEHOLDER_2}
+                                    prefix={<Icon type= {ICON_LOCK} />}
+                                    type= {PASSWORD}
+                                />
+                            )}
+                        </FormItem>
+                        <FormItem>
+                            {getFieldDecorator("remember", {
+                                initialValue: true,
+                                valuePropName: "checked",
+                            })(
+                                <Checkbox>{CHECKBOX_REMINDER}</Checkbox>
+                            )}
+
+                            <Button 
+                                type={PRIMARY} 
+                                htmlType={SUBMIT} 
+                                shape={CIRCLE} 
+                                className={CLASSNAME_LOGIN}
+                            >
+                                {TITLE}
+                            </Button>
```

Comment:
delete this button, use the `okText` on the Modal instead
```
...
okText={SUBMIT}
...
```

Reply:
Alright